quickstart/README.md

```diff
@@ -0,0 +1,81 @@
+# SRF Quickstart
+
+There are two basic ways to get started with SRF:
+- install from the conda repositories
+- pull the NGC container (available next release)
+
+#### Prerequisites
+
+- Pascal architecture or better
+- NVIDIA driver `450.80.02` or higher
+- [conda or miniconda](https://conda.io/projects/conda/en/latest/user-guide/install/linux.html)
+- if using docker:
+  - [Docker](https://docs.docker.com/get-docker/)
+  - [The NVIDIA container toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker)
+
+## Python Quickstart
+
+To get started with the SRF Python Runtime, there are several examples located in the `quickstart/examples/python` directory. These examples are organized into separate folders each showing a different topic. Each example directory has a name with the format, `XX_${EXAMPLE_NAME}`, where `XX` represents the example number (in increasing difficulty) and `${EXAMPLE_NAME}` is the example name. Below is a list of the available examples and a brief description:
+
+| #      | Name | Description |
+| ----------- | ----------- | --- |
+| 00 | SimplePipeline | A small, basic pipeline with only a single source and single sink |
+| 01 | ThreeNodePipeline | |
+| 02 | CustomTypes | |
+
+### Setup
+
+Install the SRF Python Libraries via Conda using the following:
+
+```bash
+conda install -c nvidia/label/dev srf
```

Comment:
I think we need more channels specified here. I wasn't able to install SRF with this conda command:

```
conda create --name srf-test python=3.8
conda activate srf-test
conda install -c nvidia/label/dev srf
```

Produces the following:

```
UnsatisfiableError: The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versionsThe following specifications were found to be incompatible with your system:

  - feature:/linux-64::__glibc==2.31=0
  - python=3.8 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - srf -> libsrf==22.06.00b=cuda_11.4_hcb0f886_6 -> __glibc[version='>=2.17|>=2.17,<3.0.a0']

Your installed version is: 2.31
```

Similarly, using mamba, i.e.
```
conda install -c conda-forge mamba
mamba install -c nvidia/label/dev srf
```
Errors with:
```
Encountered problems while solving:
  - nothing provides rmm 22.04.* needed by srf-22.06.00b-cuda_11.4_py38_h240fee7_6
```

Reply:
Got some conda complaints, but this seemed to work:

`conda install -c rapidsai -c nvidia/label/dev -c conda-forge srf`

Reply:
The cpp environment file has the properly ordered channel list.  Because there are 4 channels, I'd say we should use an environment file for the python as well.

Reply:
But that requires a user to clone the repo, which increases the barrier of entry.

Reply:
I suppose that's true.

I think the order should be `-c conda-forge -c rapidsai -c nvidia`, then if you want dev/nightly add `-c rapidsai-nightly -c nvidia/label/dev`

Reply:
@awthomp Can you try the quickstart guide again? I've added a lot more to the python examples and pushed new conda packages which should resolve the install error.

However, your comment "But that requires a user to clone the repo, which increases the barrier of entry." is interesting. Do you see the QSG as something that users can reference directly from Github without needing to clone the repo?

Pulling the repo will be required for running any of the hybrid or C++ examples, but might not be necessary for the python examples. I can add some setup instructions that are for the python section only that would not require cloning the repo.

The only problem I have with this is how to best organize the instructions for getting started. There are 3 use-cases I see:

1. No clone option.
   1. Limited to Python examples only
   1. User just installs the SRF conda package and can copy/paste from the examples in the README
1. Build from conda package
   1. Can run the Python/C++/Hybrid examples
   1. User installs the SRF conda package and runs a couple of commands to build the C++/Hybrid parts
1. Build from source
   1. Can run the Python/C++/Hybrid examples
   1. User builds SRF from source as normal but enables the QSG with `-DSRF_BUILD_DOCS=ON`
   1. This is necessary to test during CI and for developers

Do you have any thoughts on how to best organize all of this?

Reply:
@mdemoret-nv -- I'll take a thorough look this Monday (the 27th); I'm finishing up a family vacation today and am OOO.

My comments on cloning the repo were more focused on the installation of the SRF library. As a primary conda/Python user, I was just saying that a clone shouldn't be necessary to get a library installed. I would be totally okay with a user needing to clone a repo for examples.